### ISSUE TITLE
tapping a selected community in the LLP should deselect it

### ISSUE THREAD

**Issue by ara4n** [vector-im/riot-web#9026]
and if multiple ones are selected, it should deselect all of them

**Comment by lampholder** [comment]
Thus far the UX has modelled multiselect in something like finder - we should think carefully about any changes we make that move away from that.

**Comment by DinizCabreira** [comment]
I spent some time trying to figure out how to unselect a selected community, and eventually had to ask in the Riot matrix room to find out how (tapping on the empty space below the community list).
This may be intuitive for someone who uses Finder, but it certainly wasn't for me (and I use both several flavours of Windows and GNU/Linux).

**Comment by gnorbsl** [comment]
I first thought it is a bug, and restarted riot. It is very confusing since all rooms suddenly disappear and won't come back until you figure out how to deselect a community.

**Comment by kiliankoe** [comment]
To help with the decision, I first struggled with this as well and had to research online. Also *every single* person (~6 or so) I've introduced as a new Matrix user to Riot ran into this and could not figure it out by themselves. That's obviously not a large sample size of users, but it's 100% of the people I've seen so far 🙈

I don't see any usability issues from being able to deselect the current choice by selecting the already selected one. Multi-select still works by selecting while holding ⌘ or ⇧, as it does elsewhere in the system.
